redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -153,6 +154,35 @@ public function refreshImageDimensions()
         $this->image['height'] = $size[1];
     }
 
+    public function fixOrientation()
```

Comment:
Warum public?

Reply:
Ich dachte, ich hätte einen Use case, wo Effekte die Methode selbst aufrufen können wollten.
Aber ich habe nochmal drüber nachgedacht, und meine Überlegung war doch Quatsch, ein direkter Aufruf wäre in meinem erdachten Fall doch nicht notwendig.
Ich ändere es somit.

Diese komischen doppelten Methodenaufrufe (refreshImageDimensions und asImage) erübrigen sich dann auch, die waren auch für den Fall, dass ein Effekt die Methode selbst aufruft.